.env.example

```diff
@@ -50,6 +50,19 @@ GITHUB_TOKEN={fill me in}
 
 # ENV_WHITELIST={optional, comma separated list of env values that should be used when deploying }
 
+## EXPORT_JOB_DOWNLOADED_AGE determines how long a csv export job and file should
+## persist after it is downloaded before cleanup, default is 12.hours.
+##
+## EXPORT_JOB_MAX_AGE determines how long a csv export job and file should persist
```

Comment:
I don't see the need anymore because of customizable filters on the report.  I think 1 day is good for the default cleanup.  If 1 day is too long or short, changing the env value could be done without a code change now.